server/services/helper.service.js

```diff
@@ -0,0 +1,33 @@
+import { ErrorHandler } from '../helpers/errorHandler';
+import httpStatus from '../helpers/httpStatus';
+import { MODEL } from '../helpers/constants';
+import { getOfferById } from './offer.service';
+import { getVasRequestById } from './vasRequest.service';
+
+// eslint-disable-next-line import/prefer-default-export
+export const verifyModel = async ({ modelType, modelId }) => {
```

Comment:
Not sure why you used `verifyModel` when we are getting all the models by the Id. 

```suggestion
export const getModelById = async ({ modelType, modelId }) => {
```